packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3372,6 +3371,7 @@ export type MessageThreadProps = {
     onDisplayDateTimeString?: (messageDate: Date) => string;
     mentionOptions?: MentionOptions;
     inlineImageOptions?: InlineImageOptions;
+    enableRichTextEditor?: boolean;
```

Comment:
Should we still use `richTextEditor` name here so we won't need to change the name?